src/components/atoms/SlideOut/SlideOut.tsx

```diff
@@ -33,20 +34,28 @@ export const SlideOut = ({ children, showCloseButton = true }: SlideOutProps) =>
 
   return (
     <AnimatePresence initial={false}>
-      {currentSlideOut === "concepts" && (
+      {currentSlideOut !== "" && (
         <motion.div
           ref={ref}
           key="slideOut"
           initial={{ translateX: "-100px" }}
-          animate={{ opacity: 1, translateX: 0, transition: { duration: 0.35, ease: [0.04, 0.62, 0.23, 0.98] } }}
+          animate={{ opacity: 1, translateX: 0, transition: { duration: 0.25, ease: [0.04, 0.62, 0.23, 0.98] } }}
           exit={{ opacity: 0, transition: { duration: 0 } }}
-          className="absolute top-0 left-full h-full bg-white p-5 pb-10 min-w-[400px]"
+          className="absolute z-20 top-0 left-0 h-full bg-white p-5 pb-[70px] w-screen md:z-0 md:w-auto md:min-w-[400px] md:left-full md:pb-0"
```

Comment:
Thought for later: We should install a linting rule that orders Tailwind classes. Here `z-20` and `md:z-0` aren't next to each other which makes it harder to follow. Nothing to correct for now, just a thought.

Reply:
Open to ideas on this, personally I've found it easier to read to have the mobile classes first, then subsequent breakpoint overrides at the end, so that they are grouped by breakpoint, not individual style.